tool-plugins/nextjs-starter/src/utils/const.ts

```diff
@@ -0,0 +1,9 @@
+export const KEY_PREFIX = 'sb_ab';
```

Comment:
One question @eunjae-lee @demetriusfeijoo is this supposed to be `sb_app` or `sb_ab`? Is okay? What is `ab` in this context?

Reply:
app.. bridge... 😂